pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter_test.go

```diff
@@ -2145,6 +2145,13 @@ var _ = Describe("Validating VM Admitter", func() {
 					Field:   "spec.template.spec.domain.memory.guest",
 					Message: "Memory hotplug is only available for x86_64 and arm64 VMs",
 				}),
+				Entry("guest memory is less than 1Gi", func(vm *v1.VirtualMachine) {
```

Comment:
@acardace why did we choose to reject the spec and not follow the established route for hot plug by simply requiring a restart?

Reply:
we do require a restart, the check you highlighted only gets executed if you explicitely set `maxGuest` https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go#L513, which means you absolutely want mem-hotplug.

Reply:
We could change this, but if you specify you want memory hotplug this is an invalid VM spec.